tests/flow/test_ts_resp3.py

```diff
@@ -24,11 +24,13 @@ def test_resp3(env):
         assert res == [1000, 5.0]
         res = r.execute_command('ts.info', t1, 'DEBUG')
         res.pop(b'memoryUsage')
+        default_duplicate_policy = env.cmd("config", "get", "ts-duplicate-policy")[b"ts-duplicate-policy"]
         assert res == {
             b'totalSamples': 1000,
             b'firstTimestamp': 1, b'lastTimestamp': 1000,
             b'retentionTime': 0, b'chunkCount': 2, b'chunkSize': 128,
-            b'chunkType': b'compressed', b'duplicatePolicy': None,
+            b'chunkType': b'compressed',
+            b'duplicatePolicy': default_duplicate_policy,
```

Comment:
not that it particularly matters here, but for the future: when writing RLTests, passing the argument `decodeResponses=True` into `Env()` will obviate the need to use `b`-strings.

Reply:
Well, 
![image](https://github.com/user-attachments/assets/63f0aecf-db8e-499b-ac3b-0f211ffcfec2)

Reply:
oh i know very well. it was like that when i got here, too.